libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -188,6 +195,49 @@ class NavigationView : ConstraintLayout {
         updateNavigationViewOptions(navigationViewOptions)
     }
 
+    private val configurationOnStyleLoadListener = OnStyleLoadedListener {
+        removeConfigureOnStyleLoadListener()
+        mapView.location.setLocationProvider(navigationLocationProvider)
+        mapView.location.enabled = true
+
+        MapboxDropInUtils.getLastLocation(
+            this@NavigationView.context,
+            WeakReference<(Expected<Exception, LocationEngineResult>) -> Unit> { result ->
+                result.fold(
+                    {
+                        Log.e(TAG, "Error obtaining current location", it)
+                    },
+                    {
+                        it.lastLocation?.apply {
+                            cameraViewModel.consumeLocationUpdate(this)
+                            navigationLocationProvider.changePosition(this, listOf(), null, null)
+                            initLocationComponentPuck()
```

Comment:
This should happen regardless of whether we manage to obtain the initial location or not.

If we would fail initially, the puck would never show up afterwards.

Reply:
If you look at `LocationPuckManager::initialize` line 58 in the Maps code, if `lastlocation` is null then a `hide()` method gets called and the puck won't display.  It appears that we can only set the puck when there is a location.  If I move the call to `initLocationComponentPuck()` as you suggest where would it go?

Reply:
I noticed that once a trip session starts the navigation puck appears.  I"ll close this PR.